puente-analytics-service/lambdas/etl/Makefile

```diff
@@ -20,4 +20,4 @@ invoke:
 # Run the application locally
 .PHONY: run
 run:
-	sudo DOCKER_HOST=unix:///Users/hopetambala/.docker/run/docker.sock sam local start-lambda --template ../../templates/cloudformation-master.yaml --debug
\ No newline at end of file
+	DOCKER_HOST=unix:///Users/hopetambala/.docker/run/docker.sock sam local start-lambda --template ../../templates/cloudformation-dev.yaml --debug --env-vars env.json 
```

Comment:
This file is meant to make your life easier @awalsh272 . Let me know if you need help getting it up and running (I don't know how familiar you are with make files)